slices/libcurl4.yaml

```diff
@@ -0,0 +1,20 @@
+# Easy-to-use client-side URL transfer library (OpenSSL flavour)
+package: libcurl4
+
+slices:
+  libs:
+    essential:
+      - libbrotli1_libs
+      - libc6_libs
+      - libgssapi-krb5-2_libs
+      - libidn2-0_libs
+      - libldap-2.5-0_libs
+      - libpsl5_libs
+      - libssh2-1_libs
```

Comment:
This should be [libssh-4](https://packages.ubuntu.com/jammy/libssh-4) actually. See: https://packages.ubuntu.com/jammy/libcurl4